test/benchmarks_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:convert' show JsonEncoder;
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+import 'package:gallery/benchmarks/common.dart';
+
+import 'package:web_benchmarks/server.dart';
+
+final listOfBenchmarks = <String>[
```

Comment:
testing a benchmark test... nice 👍

Reply:
Thank you! I borrowed this test from the `web_benchmarks` test itself.